model/state_model_nlparam.jl

```diff
@@ -49,6 +49,8 @@ bmkyr=2016
 
 #sld is the slide dictionary of benchmark values filtered for benchmark year
 sld = Dict(k => convert_type(Dict, dropzero(filter_with(d[k], (yr = bmkyr,); drop = true))) for k in keys(d))
+# zld = Dict(k => convert_type(Dict, fill_zero(set,
```

Comment:
This could look something like this if we go the zeros route instead of haskey.
Pros: Consistent with calibration routine. Would eliminate need for haskey/errors that would arise if any values are set to zero.
Possible con: **Would we loose significant speed-ups from using sparse arrays in JuMP?**

Reply:
Yes that could work as well with ```fillzero```. Would not eliminate need for ```haskey``` in equation/variable filtering, but could eliminate the use of ```get``` when loading benchmark values into nlparameters.